src/plugins/plugin.decimation.js

```diff
@@ -63,22 +78,32 @@ function lttbDecimation(data, availableWidth, options) {
   }
 
   // Include the last point
-  decimated[sampledIndex++] = data[data.length - 1];
+  decimated[sampledIndex++] = data[endIndex];
 
   return decimated;
 }
 
-function minMaxDecimation(data, availableWidth) {
+function minMaxDecimation(data, start, count, availableWidth) {
   let avgX = 0;
   let countX = 0;
-  let i, point, x, y, prevX, minIndex, maxIndex, startIndex, minY, maxY;
+  let i,
```

Comment:
We have the undefined variables on single line everywhere, so it would be nice to keep consistent. Your editor probably changed this?

Reply:
My mistake, same for comma, will fix it